src/main/java/games/strategy/engine/framework/GameDataUtils.java

```diff
@@ -37,7 +37,8 @@ public static GameData cloneGameData(final GameData data, final boolean copyDele
    * Translate units,territories and other game data objects from one
    * game data into another.
    */
-  public static Object translateIntoOtherGameData(final Object object, final GameData translateInto) {
+  @SuppressWarnings("unchecked")
```

Comment:
My only suggestion would be to keep the effect of a `@SuppressWarnings` as tight as possible.  In this case, applying it to a temporary variable where the actual unchecked cast occurs, so as not to preclude detecting other unchecked warnings anywhere else in this method in the future:

```java
try {
  @SuppressWarnings("unchecked")
  final T otherObject = (T) in.readObject();
  return otherObject;
} catch (final ClassNotFoundException ex) {
  // should never happen
  throw new RuntimeException(ex);
}
```